controllers/controller.js

```diff
@@ -42,3 +44,32 @@ exports.getCommentsByReviewId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+  const newReview = {
+    body: req.body.body,
+    votes: 0,
```

Comment:
Votes should default to zero (because of sql) when you post the comment, so you shouldn't have to do this in the controller.